components/myforms/HTMLDownload/ProtectedWarning.tsx

```diff
@@ -3,20 +3,35 @@ import { useTranslation } from "next-i18next";
 
 // Note: use lang prop if you want to force a specific language e.g. HTMLDownload component
 interface ProtectedWarningProps {
+  securityAttribute: string;
   lang?: string;
 }
 
 export const ProtectedWarning = (props: ProtectedWarningProps) => {
   const { t, i18n } = useTranslation(["my-forms"]);
   const { lang = i18n.language } = props;
+
+  const renderSecurityAttribute = () => {
+    switch (props.securityAttribute) {
```

Comment:
Something for now or maybe a future PR

In SetResponseDelivery.tsx we have:

```
const classificationOptions = [
  { value: "Unclassified", en: "Unclassified", fr: "Non classifié" },
  { value: "Protected A", en: "Protected A", fr: "Protégé A" },
] as const;
```

So we should figure out how we want to implement so we don't duplicate / have one place to update as need be.

Reply:
Ya - we can make another PR to update classificationOptions to use the lang files.